platform/local/nsexec.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package local
+
+import (
+	"os/exec"
+
+	"github.com/vishvananda/netns"
+
+	"github.com/coreos/mantle/util"
+)
+
+type NsCmd struct {
+	util.ExecCmd
+	NsHandle netns.NsHandle
+}
+
+func NewNsCommand(ns netns.NsHandle, name string, arg ...string) util.Cmd {
+	return &NsCmd{
+		ExecCmd:  util.ExecCmd{exec.Command(name, arg...)},
+		NsHandle: ns,
+	}
+}
+
+func (cmd *NsCmd) CombinedOutput() ([]byte, error) {
```

Comment:
You could probably encapsulate this pattern in a helper function and reduce this to `return execWithinNS(cmd.ExecCmd.CombinedOutput())`.